maps/away/ships/elyra/elyra_strike_craft/elyra_strike_craft.dm

```diff
@@ -1,7 +1,7 @@
 /datum/map_template/ruin/away_site/elyran_strike_craft
```

Comment:
the old current elyra strike craft should be deleted, as the ship in this PR is effectively its superior replacement:
1. there shouldn't be two ships of the same faction, filling the same role
2. the old elyra strike craft is inferior and does not follow current mapping standards (it does not have atmos for example)